bin/checks.ts

```diff
@@ -0,0 +1,159 @@
+#!/usr/bin/env node
+/* eslint-disable @typescript-eslint/camelcase */
+
+import args from 'args'
```

Comment:
Why not using yargs instead of importing a new libs, this also have some limitations so that's why we changed to yargs.

Reply:
Also yargs, generates typescript types for you, look at the examples in this repo and node-setup